src/components/foundation/BodyExample.js

```diff
@@ -1,39 +1,43 @@
-import React from 'react';
+import React from "react";
 
 const headingExampleStyles = {
-	borderTop: '1px solid lightgray',
-	padding: '20px'
+  borderTop: "1px solid lightgray",
```

Comment:
did anything apart from formatting change here?

Reply:
that page was using sidebar instead of dg_sidebar and that was all that was manually changed. Any other changes must be formatting.